docs/content/preview/yugabyte-platform/troubleshoot/install-upgrade-yp-issues.md

```diff
@@ -64,3 +64,23 @@ do
   ssh $IP firewall-cmd --zone=public --add-port=6379/tcp;
 done
 ```
+
+## Configure load balancer for Helm charts
+
+If you experience issues with configuring a load balancer when installing and configuring YugabyteDB Anywhere on Kubernetes, you should define the Cross-Origin Resource Sharing (CORS) domain configuration by setting the [additionAllowedCorsOrigins](https://github.com/yugabyte/charts/blob/master/stable/yugaware/values.yaml#L66) value to the new domain involved.
+
+For Amazon Web Services (AWS) load balancers, the default service configuration is adequate; however, if customization is required, the backend protocol should be configured to HTTP by setting the following AWS load balancer controller annotations:
+
+- `aws-load-balancer-scheme` can be set to `internal` or `internet-facing` string value.
+- `aws-load-balancer-backend-protocol` and `aws-load-balancer-healthcheck-protocol` should be set to the `http` string value.
+
+Consider the following sample configuration:
+
+```properties
+service.beta.kubernetes.io/aws-load-balancer-type: "ip"
+service.beta.kubernetes.io/aws-load-balancer-scheme: "internet-facing"
+service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "http"
+service.beta.kubernetes.io/aws-load-balancer-healthcheck-protocol: "http"
+```
+
+For YugabyteDB Anywhere HTTPS configuration, you should set your own key or certificate. If you do provide this setting, the default public key is used, creating a potential security issue.
```

Comment:
We actually have a proposal to fix this, so this will hopefully be a non-issue in a few weeks, we could hold off on this third point.

Reply:
Commented this out.